src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -23,274 +23,272 @@
+        
+        public int chunksPerTicket(@Nullable String modid) {
+            return getByMod(chunksPerTicket, "chunksPerTicket", modid);
         }
     }
+    
+    public static final ForgeConfigSpec chunk_spec = CHUNK_BUILDER.build();
 
-    public static class CLIENT {
-        public static boolean forgeCloudsEnabled() {
-            return ForgeConfig.INSTANCE.configData.<Boolean>getOrElse("general.forgeCloudsEnabled", true);
-        }
+    private static void loadFrom(final Path configRoot) {
+        Path configFile = configRoot.resolve("forge.toml");
+        spec.setConfigFile(configFile);
+        LogManager.getLogger().debug(CORE, "Loaded Forge config from {}", configFile);
+
+        configFile = configRoot.resolve("forge_chunks.toml");
+        chunk_spec.setConfigFile(configFile);
+        LogManager.getLogger().debug(CORE, "Loaded Forge Chunk config from {}", configFile);
     }
 
-    public static class CHUNK {
-        public static boolean enableModOverrides() {
-            return ForgeConfig.INSTANCE.chunkData.<Boolean>getOrElse("defaults.enable", true);
-        }
-        public static int playerTicketCount() {
-            return ForgeConfig.INSTANCE.chunkData.<Integer>getOrElse("defaults.playerTicketCount", 500);
-        }
-        public static int dormantChunkCacheSize() {
-            return ForgeConfig.INSTANCE.chunkData.<Integer>getOrElse("defaults.dormantChunkCacheSize", 0);
-        }
-        private static int maxTickets() {
-            return ForgeConfig.INSTANCE.chunkData.<Integer>getOrElse("defaults.maxTickets", 200);
-        }
-        public static int chunksPerTicket() {
-            return ForgeConfig.INSTANCE.chunkData.<Integer>getOrElse("defaults.chunksPerTicket", 0);
-        }
-        public static int maxTickets(@Nullable String modid) {
-            if (!enableModOverrides() || modid == null)
-                return maxTickets();
-            Map<Object,Object> data = ForgeConfig.INSTANCE.chunkData.<List<Map<Object, Object>>>getOrElse("mods", Collections.emptyList())
-                    .stream().filter(e -> modid.equals(e.get("modid"))).findFirst().orElse(null);
-            Integer ret = data == null ? null : (Integer)data.get("maxTickets");
-            return ret == null ? maxTickets() : ret;
-        }
-        public static int chunksPerTicket(@Nullable String modid) {
-            if (!enableModOverrides() || modid == null)
-                return chunksPerTicket();
-            Map<Object,Object> data = ForgeConfig.INSTANCE.chunkData.<List<Map<Object, Object>>>getOrElse("mods", Collections.emptyList())
-                    .stream().filter(e -> modid.equals(e.get("modid"))).findFirst().orElse(null);
-            Integer ret = data == null ? null : (Integer)data.get("chunksPerTicket");
-            return ret == null ? chunksPerTicket() : ret;
-        }
+    public static void load() {
+        loadFrom(Paths.get("config"));
```

Comment:
Should this use `FMLPaths.CONFIGDIR` instead as the game directory may not be the same as the working directory?